string full_plan_filepath
---
int64 success_bool